kentik_api_library/tests/e2e/synthetics/test_agent.py

```diff
@@ -19,7 +19,7 @@
 
 
 @pytest.mark.skipif(not credentials_present, reason=credentials_missing_str)
-def test_agent_test_crud() -> None:
+def test_agent_test_crud(test_labels) -> None:
```

Comment:
type hints missing

Reply:
here and in other tests

Reply:
This is actually reference to a pytest fixture. I don't think it can be effectively type hinted. Unless you know the correct solution, I'm inclined to leave it as is.